source/standards/disaster-recovery.html.md.erb

```diff
@@ -0,0 +1,77 @@
+---
+title: Disaster Recovery
+last_reviewed_on: 2021-07-26
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+The [GOV.UK Service Manual](https://www.gov.uk/service-manual) describes how you should [design your service to maximise uptime](https://www.gov.uk/service-manual/technology/uptime-and-availability-keeping-your-service-online).
+However, it is inevitable that your service will experience unscheduled downtime at some point, which could have a serious impact on users. You must plan your response for when this happens.
+
+## Planning for disaster recovery
+
+In digital services, a disaster is a significant event that either disrupts your service or completely prevents you from providing it. A disaster can have various causes, including:
+
+- a cloud provider suffering a serious regional outage
+- accidental or malicious destruction
+
+A disaster recovery plan details how you will recover from these serious incidents.
+
+### Understand risks and threats to your service
+
+You should work with the [Information Assurance (IA)](https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/operations/information-assurance) and [Cyber Security](https://gds-way.cloudapps.digital/standards/cdio-pillars.html#cdio-security) teams to understand the risks to your service. This will help you build a more resilient and secure digital service.
+
+You should also work with risk and service owners to plan for the worst-case scenarios. This is particularly important for your data, as loss or theft of data is disastrous for most services.
+
+Make sure you consider how [disruption to or unavailability of your dependencies](https://www.gov.uk/service-manual/technology/uptime-and-availability-keeping-your-service-online#issues-that-can-affect-uptime) can affect your service. These dependencies can include obvious systems, such as your hosting provider, but also less obvious systems such as your:
+
+- Domain Name System (DNS) provider
+- Content Delivery Network (CDN)
+- any Virtual Private Networks (VPNs) your development team might use to access production systems
+build pipelines
+- external library and container registries (for example, RubyGems, PyPI or Docker Hub)
+- secrets management providers
+
+### Back up your data
+
+You should back up your data. How often you back up will depend on your [Recovery Point Objective (RPO)](https://cloud.google.com/architecture/dr-scenarios-planning-guide#basics_of_dr_planning). An RPO determines what you consider an acceptable loss of data between the last backup and the point of data loss. For example, if you back up your data nightly, your RPO is 24 hours, the length of time between the backups. In practice, many modern database service cloud providers provide [point-in-time recovery](https://docs.aws.amazon.com/aws-backup/latest/devguide/point-in-time-recovery.html) from backups. This can result in an RPO of around 5 minutes or less.
```

Comment:
The description of RPOs is sensible, but the examples are backwards (cause vs. effect). An RPO is an objective that you want to meet, and you make choices based on that. So rather than "if you back up your data nightly, your RPO is 24 hours" it should be "if your RPO is 24 hours, you might choose to take nightly backups." Likewise,  point-in-time recovery ... can result in an RPO of around 5 minutes or less" should be "if you determine that you need a 5 minute RPO, then point-in-time snapshotting and recovery might be necessary."

Reply:
Ah yes! I totally forgot about the thread on this in Slack. I'll have a go at a reword